charts/nginx/templates/nginx-configmap.yaml

```
@@ -85,14 +87,16 @@ data:
 
         -- Split the query string into individual parameters
         for param in string.gmatch(queryString, "([^&]+)") do
-          local paramName, paramValue = string.match(param, "(.+)=(.+)")
+          local paramName, paramValue = string.match(param, "(.+)=(.*)") -- Allow empty paramValue
```

Shouldn't this be consistent between the two blocks?
```suggestion
          local paramName, paramValue = string.match(param, "([^=]+)=(.*)") -- Allow empty paramValue
```